estate/models/offer.py

```diff
@@ -0,0 +1,47 @@
+from odoo.exceptions import UserError
+from odoo import fields, models, api
+from odoo.tools import add
+
+
+class Offer(models.Model):
+    _name = "offer"
+    _description = "Offer"
+    price = fields.Float(string="Price")
+    status = fields.Selection(copy=False, string="Status", selection=[("accepted", "Accepted"), ("refused", "Refused")],
+                              required=True)
+    partner_id = fields.Many2one("res.partner", string="Partner", required=True)
+    property_id = fields.Many2one("estate_property", string="Estate Property", required=True)
+    validity = fields.Integer(string="Validity", default="7")
```

Comment:
`string` could be clearer: what is the Unit of Measure? Month?
Also, the default should be of the good type, so 7, not "7"